addons/stock/models/stock_move_line.py

```diff
@@ -492,7 +492,7 @@ def _action_done(self):
         # Now, we can actually move the quant.
         done_ml = self.env['stock.move.line']
         for ml in self - ml_to_delete:
-            if ml.product_id.type == 'product':
+            if ml.product_id.type in ['product', 'consu']:
```

Comment:
In case of consu without a package_id/dest_package_id, you should bypass the `_compute_quantity` and the `_update_available_quantity`, no?